docs/app/Components/ComponentDoc/ComponentProps.js

```diff
@@ -65,6 +84,64 @@ export default class ComponentProps extends Component {
     )
   }
 
+  expandEnums = (value) => {
+    const parts = value.split('.')
+    if (parts[0] === 'SUI') {
+      return SUI[parts[1]]
+    } else if (parts[1] === '_meta') {
+      // these look like Sidebar._meta.props.animation
+      return _.get(this.props, `meta.props.${parts[3]}`, value)
```

Comment:
`_meta.props` is in the process of going away and the values are being inlined into the `propTypes` checkers, [example](https://github.com/Semantic-Org/Semantic-UI-React/pull/1216/files#diff-234ffdf68307872a89a512e653bea71aL29).  You can see progress [here](https://github.com/Semantic-Org/Semantic-UI-React/issues/524).  Given this, we're removing all traces of `_meta.props` from the codebase.

In this case, we should probably just not parse the `meta.props` here at all, for now.  Most the enums will show up automatically because the values can be parsed by react-docgen.  Those that cannot will be those that have more complicated checkers, which, we'll have to solve another way.